### PR TITLE
renaming object, so it has more meaningful name and it's easier to debug 

### PR DESCRIPTION
**Pull request by wejdross**
## Summary

* Short summary of what's included in the PR
* Give special note to breaking changes

## Checklist

- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog
- [x] Update tests.
- [x] Link this PR to related issues.

### CODE REVIEW

File: pkg/comp-functions/functions/vshn-postgres-func/loadBalancer.go [hunk 1]
```diff
@@ -55,15 +55,15 @@ func AddLoadBalancerIPToConnectionDetails(ctx context.Context, iof *runtime.Runt
 		k8sservice.Spec.Type = v1.ServiceTypeClusterIP
 	}
 
-	if err := iof.Desired.PutIntoObject(ctx, k8sservice, fmt.Sprintf("%s-%s", comp.GetName(), serviceName)); err != nil {
+	if err := iof.Desired.PutIntoObject(ctx, k8sservice, fmt.Sprintf("%s-%s", comp.GetName(), "service")); err != nil {
```
Comment (Kidswiss) [inline pkg/comp-functions/functions/vshn-postgres-func/loadBalancer.go:58]: Maybe call it `masterservice` or `mainservice`?

If customers request it, we might also expose the slaves/replicas/secondaries in the future as well.

**Review by Kidswiss — CHANGES_REQUESTED** [review]
Just one nitpick.

File: pkg/comp-functions/functions/vshn-postgres-func/loadBalancer.go (hunk 1 above)
Comment (wejdross) [reply pkg/comp-functions/functions/vshn-postgres-func/loadBalancer.go]: I just renamed it to `primary-service` to comply with Your idea and @TheBigLee suggestions

**Review by Kidswiss — APPROVED** [review]

**Comment by wejdross** [comment]
@Kidswiss @TheBigLee I forgot to update this `postgresql-master`, I just pushed commit that fixes that, so both object and underlying service name reuse the same name `primary-service`

This is how it looks like now:
```
Service:
primary-service                  LoadBalancer   10.96.192.252   172.18.0.200 5432:30160/TCP      44s
Object:
postgresql-prod-ww4mc-primary-service              Service             kubernetes       True     True    3m1s
```
wdyt?

**Review by Kidswiss — APPROVED** [review]
LGTM

Please adjust any already written docs with the new service name though.

**Pull request merged**